execution/setStackContext.sh

```diff
@@ -188,14 +188,14 @@ if [[ ! -f "${CF_DIR}/${TEMPLATE}" ]]; then
 fi
 
 # Permit renaming of stack files without affecting existing stack status
-if [[ -f "${CF_DIR}/${STACK}" && -n "${AWSID}" ]]; then
-    STACK_NAME=$(jq -r ".Stacks[0].StackName" <  "${CF_DIR}/${STACK}")
-fi
-
-# Permit renaming of stack files without affecting existing stack status
-if [[ -f "${CF_DIR}/${STACK}" && -n "${AZID}" ]]; then
-    STACK_NAME=$(jq -r ".properties.outputs.resourceGroup.value" <  "${CF_DIR}/${STACK}")
-fi
+case ${ACCOUNT_PROVIDER} in
```

Comment:
Looks like we've lost the check that the stack already exists - this processing should only apply if the stack exists.

Reply:
@ml019 

fixed to check for the stack. with commit #73386a7